src/services/logging/formatters.ts

```diff
@@ -0,0 +1,96 @@
+import * as Hapi from 'hapi';
+import * as Cookie from 'cookie';
+import * as JWT from 'jsonwebtoken';
+import { compose } from 'ramda';
+import { RequestErrorEvent, RequestSentEvent, LogLine } from './types';
+import { getConfig } from '../../../config';
+
+
+const { auth: { standard } } = getConfig(process.env.NODE_ENV);
+
+
+const tagOrder: (keyof LogLine)[] = [
+  'event',
+  'method',
+  'path',
+  'statusCode',
+  'origin',
+  'userId',
+  'organisationId',
+  'query',
+  'payload',
+  'response',
+];
+
+const errorOrder: (keyof LogLine)[] = ['errorMessage', 'errorStack'];
+
+const extractCookie = (c: string) => {
+  const { [standard.cookie.name]: token } = Cookie.parse(c);
+  return token;
+};
+
+const decodeToken = (c: string): { userId?: number, organisationId?: number } => {
+  let decoded: { userId?: number, organisationId?: number };
```

Comment:
Do we have a type for cookie? Also assuming that these values should not be optional.

Reply:
They are optional because if the token is invalid, `verfiy` will throw and we simply return an empty object (because we still want the logging to continue).

A little hacky in terms of the types but I just want logs now, I can tidy it up later.